packages/datadog-plugin-next/test/index.spec.js

```diff
@@ -38,6 +38,7 @@ describe('Plugin', function () {
           execSync('node build', {
             cwd: __dirname,
             env: {
+              ...process.env,
```

Comment:
Why is this also needed?

Reply:
Using the `env` field anywhere in the child process API _replaces_ the env, it _does not_ mix it in. This means it may break your shell if it relies on certain environment variables being present. In my case, it was unable to find `node` in the `PATH` without it, because there _was_ no `PATH` set.